### PR TITLE
feat: reparse the file after a content change

### PR DESCRIPTION
**Pull request by alacambre-yseop**

### CODE REVIEW

File: server/src/server.ts [hunk 1]
```diff
@@ -80,8 +80,7 @@ connection.onHover((_params) => {
 });
 
 const validateTextDocumentOnEvent = (event: TextDocumentChangeEvent) => validateTextDocument(event.document);
-documents.onDidOpen(validateTextDocumentOnEvent);
-documents.onDidSave(validateTextDocumentOnEvent);
+documents.onDidChangeContent(validateTextDocumentOnEvent);
```
[the remark this answers is not in the document]
Comment (alacambre-yseop) [reply server/src/server.ts]: Yes.

**Comment by mminot-yseop** [comment]
I'll test that to make sure there are no huge lags when rapidly editing the file xD

**Comment by mminot-yseop** [comment]
Seems OK but I got flooded with…

> INVALID text edit -> must have at least a label
extensionHostProcess.js:570

What's this?

**Comment by alacambre-yseop** [comment]
> Seems OK but I got flooded with…
> 
> > INVALID text edit -> must have at least a label
> > extensionHostProcess.js:570
> 
> What's this?

I don't know. A fullscreen screenshot would be appreciated.

**Comment by mminot-yseop** [comment]
I think I noticed that _after_ my tests xD
I'll try to give it another go.

**Comment by alacambre-yseop** [comment]
Maybe you tried to use the quick-fix feature while we don't provide it?

**Pull request merged**